app/services/trainees/create_from_apply.rb

```diff
@@ -30,7 +30,7 @@ def call
       save_personal_details!
       create_degrees!
       application_record.imported!
-
+      Trainees::SetAcademicCycles.call(trainee: trainee)
```

Comment:
I think you'll probably need to save the trainee as the service doesn't do that, it just sets the academic cycles. The tests aren't picking this up.